src/components/MoneyTemporaryForRefactorRequestConfirmationList.js

```diff
@@ -493,17 +493,18 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
     // Auto select the tag if there is only one enabled tag and it is required
     useEffect(() => {
         let updatedTagsString = '';
+        const isMultipleLevelsTags = policyTagLists && policyTagLists.length > 1;
         policyTagLists.forEach((tagList, index) => {
             const enabledTags = _.filter(tagList.tags, (tag) => tag.enabled);
-            if (enabledTags.length !== 1 || !tagList.required || TransactionUtils.getTag(transaction, index)) {
+            if (enabledTags.length !== 1 || isMultipleLevelsTags ? !tagList.required : !isTagRequired || TransactionUtils.getTag(transaction, index)) {
```

Comment:
```suggestion
            if (enabledTags.length !== 1 || !tagList.required || TransactionUtils.getTag(transaction, index)) {
```

Looking at this again, I think `isTagRequired` probably should/will be depreciated anyway as now all tags (including single level) will have the `required` property.

Reply:
Haha oh sorry I responded on the issue first before seeing this. I'd vote for proceeding with the expectation that single-level tags will also have the `required` field set as the `isTagRequired` field seems obsolete to me. cc @puneetlath in case you have thoughts. (And sorry for all the pings!)

Reply:
Asked here: https://expensify.slack.com/archives/C01GTK53T8Q/p1709227512388819?thread_ts=1708732718.173419&cid=C01GTK53T8Q